.gitignore

```diff
@@ -2,6 +2,7 @@
 webapp/static/vendor/jquery.powertip-1.2.0/
 node_modules/
 webapp/__pycache__/
+webapp/data_access/__pychache__
```

Comment:
I asked you to write a "general" rule to ignore all __pycache__ folder no matter where they are in the project.